apiSpec/generateTypes.js

```diff
@@ -96,9 +96,9 @@ function cleanClasses(updatedContent) {
 
 function addGeneratedHeader(updatedContent) {
   const header = `/**
- * Copyright 2024-present Coinbase Global, Inc.
+ * Copyright 2025-present Coinbase Global, Inc.
```

Comment:
Is there any way to only apply this to new stuff? or should we just always leave it at 2024-present since the original SDK was released in 2024?

Reply:
i look to it as generated in X year. if you pull older npm versions, it would have the original date